autorest/codegen/models/code_model.py

```diff
@@ -33,6 +34,9 @@ def get_serialization_type(self):
     def get_python_type(self, namespace=None):
         return namespace+"FAKEDOCSTRING"
 
+    def get_python_type_annotation(self) -> str:
+        return self.get_python_type()
```

Comment:
Actually, let's remove this `FakeSchema` totally, as I said in the comment it's uncessary. Let's the lookup raise if it doesn't find the mapping